packages/loader/container-loader/src/container.ts

```diff
@@ -277,6 +277,9 @@ export interface IPendingContainerState {
 
 const summarizerClientType = "summarizer";
 
+/**
+ * @deprecated - In the next release Container will no longer be exported, IContainer should be used in its place.
```

Comment:
These are kind of unusual markings since they're only "deprecated" for public use - not sure if this is the appropriate marking vs. `@internal`?  I'm fine either way, mainly an academic question.

I assume either way the marking gets removed in `next` (maybe wait to merge that PR until after this change gets merged to `next` so you can take the markings back off again?

Reply:
saw this after i merged, but can do. i deprecated, as to external consumer the class is deprecated. we can't switch to internal mid release, as internal allows breaking change, so we can only switch at major release boundaries.